platform/darwin/src/MGLStyle.h

```diff
@@ -564,6 +565,8 @@ MGL_EXPORT
  */
 - (void)removeImageForName:(NSString *)name;
 
+@property (nonatomic, strong) MGLLight *light;
```

Comment:
This property is currently under the `#pragma mark` for style images. It should get its own mark. jazzy turns marks into section headings.